sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 2.0.0-beta.3 (Unreleased)
 
+- Updated the `@azure/msal-node` dependency to version `^1.0.2`, which fixes the `DeviceCodeCredential` cancelation.
```

Comment:
"fixes the `DeviceCodeCredential` cancelation" does not tell what the original problem was.

```suggestion
- Updated the `@azure/msal-node` dependency to version `^1.0.2`, which allows cancelling of an ongoing `getToken()` operation on `DeviceCodeCredential`.
```